.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "4.2"
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  - npm install -g selenium-standalone
+  - selenium-standalone install
+  - selenium-standalone start > /dev/null 2>&1 &
```

Comment:
On the note of "a more recognized selenium runner", protractor has almost 10x the amount of downloads =/

https://www.npmjs.com/package/protractor

https://www.npmjs.com/package/selenium-standalone